services/data-info/src/data_info/services/stat.clj

```diff
@@ -3,6 +3,7 @@
             [clojure.tools.logging :as log]
             [dire.core :refer [with-pre-hook! with-post-hook!]]
             [slingshot.slingshot :refer [throw+]]
+            [medley.core :only [remove-vals]]
```

Comment:
Good point. It looks like this was the only thing using medley in data-info at all, actually, so I've also removed it from project.clj